src/components/button/SCreateButton.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import { IconType } from "react-icons";
+import React from "react";
+import { ButtonSize } from "../../styles/theme";
```

Comment:
```suggestion
import { ButtonSize } from "@/styles/theme";
```
절대 경로 적용해주시면 좋을 것 같습니당